service/platform/index.js

```diff
@@ -205,6 +221,8 @@ class ServerPlatform extends Tp.BasePlatform {
             this._pulse = new PulseAudio();
             this._pulse.on('error', (err) => { console.error('error on PulseAudio', err); });
 
+            this._ensurePulseConfig();
```

Comment:
Should we have a .catch here? In case loading modules fail, we don't want to crash.

Reply:
yeah, added just to be sure we don't crash with modules listing. Loading failure doesn't throw an exception.